client/post-editor/editor-title/index.jsx

```diff
@@ -101,19 +104,36 @@ export default React.createClass( {
 				}
 				<TrackInputChanges onNewValue={ this.recordChangeStats }>
 					<TextareaAutosize
-						{ ...omit( this.props, Object.keys( this.constructor.propTypes ) ) }
+						{ ...ownProps }
 						className="editor-title__input"
-						placeholder={ this.translate( 'Title' ) }
+						placeholder={ translate( 'Title' ) }
 						onChange={ this.onChange }
 						onInput={ this.resizeAfterNewlineInput }
 						onBlur={ this.onBlur }
 						autoFocus={ isNew && ! isMobile() }
 						value={ post ? post.title : '' }
-						aria-label={ this.translate( 'Edit title' ) }
+						aria-label={ translate( 'Edit title' ) }
 						ref="titleInput"
 						rows="1" />
 				</TrackInputChanges>
 			</div>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, ownProps ) => {
+		const site = getSelectedSite( state );
+		const editedPostId = getEditorPostId( state );
+		const post = getEditedPost( state, site.ID, editedPostId );
+		const isNew = isEditorNewPost( state );
+
+		return {
+			isNew,
+			post,
+			site,
+			ownProps
```

Comment:
why are we adding `ownProps` here? it will come in as `this.props.ownProps` - is that the intention? also, did you know that `ownProps` automatically gets merged into the props through `connect` even if we never use them?